gatsby-theme-landing-page/src/components/markdown-text.js

```diff
@@ -42,7 +43,7 @@ export default function MarkdownText({
 
   const shouldUseInline = !!as;
   const sanitizeOptions = shouldUseInline ? inlineOptions : blockOptions;
-  const html = childMarkdownRemark.html.replaceAll(newLineRegex, "<br />");
```

Comment:
Making paragraphs render like normal here; not sure if this was used for anything else